app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -1,6 +1,15 @@
 class TreeBuilderChargebackRates < TreeBuilderChargeback
   private
 
+  def tree_init_options
+    super.merge!(:full_ids => false, :click_url => "/chargeback_rate/tree_select/", :onclick => "miqOnClickGeneric")
+  end
+
+  # Get root nodes count/array for explorer tree
+  def x_get_tree_roots
+    super.map { |type| type.merge!(:selectable => false) }
```

Comment:
You don't really need to run an in-place merge:
```suggestion
    super.map { |type| type.merge(:selectable => false) }
```